libs/sdk/testing/src/lib/matchers/matchers.spec.ts

```diff
@@ -220,7 +222,7 @@ describe('Jasmine matchers', () => {
 
       it('should allow configuration override', waitForAsync(() => {
         const element = createFailingElement();
-        expect(element).toBeAccessible(() => {}, {
+        expect(element).toBeAccessible(undefined, {
```

Comment:
Why is the `undefined` required? Is this an intentional change?

Reply:
Both parameters here are optional. The first is a callback function and the second is a config. We were actually only testing the second though and eslint doesn't like the empty function so I just changed it to `undefined` so that the warning would go away.